docs/using/selecting.md

```diff
@@ -267,7 +267,7 @@ Whenever a docker image is pushed to the container registry, it is tagged with:
 - the `latest` tag
 - a 12-character git commit SHA like `1ffe43816ba9`
 - a date formatted like `2023-01-30`
-- OS version like `ubuntu-22.04`
+- OS version like `ubuntu-24.04`
```

Comment:
Let's keep this line the same, because it corresponds to the tag mentioned above and we don't actually need to update it